PrototopeTestApp/JSTest.swift

```diff
@@ -16,5 +16,7 @@ func makeJSLayer() {
 		let lineNumber = value.objectForKeyedSubscript("line")
 		println("Exception on line \(lineNumber): \(value)")
 	}
-	println(context.evaluateScript("var layer = new Layer({parent: Layer.root}); Layer.animate({duration: 3.0, curve: AnimationCurve.EaseOut, animations: function() { layer.x = 400 }}); layer.gestures = [new PinchGesture({handler: function(phase, sequence) {console.log(sequence.currentSample.scale)}})]; layer.backgroundColor = new Color({red: 0.5, green: 0.7, blue: 0.1, alpha: 0.7}); layer.frame = new Rect({x: 75, y: 80, width: 400, height: 400}); layer.border = new Border({color: Color.black, width: 2}); layer.shadow = new Shadow({alpha: 1.0}); (new Sound({name: 'Glass'})).play()"))
+    
+    let script = NSString(contentsOfURL: NSBundle.mainBundle().URLForResource("JSTest", withExtension: "js")!, encoding: NSUTF8StringEncoding, error: nil)
```

Comment:
omg thank you for doing this

I was always just one notch too lazy to bother. :)